### PR TITLE
docs(next): update init-deployment to reflect changes in structure

### PR DESCRIPTION
**Pull request by MetecanAydin**
5755 📲 What

Update init-deployment to reflect changes made to deployment code structure.

#### 🤔 Why
		
Reflect new codebase updates and behaviour
		
#### 🛠 How
		
Update docs

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: docs/nx/next/init-deployment.md [hunk 1]
```diff
@@ -87,38 +90,40 @@ As a rule of thumb, each task here references a target execution via Nx defined
...
 
 ```yaml
-"helm-lint":
+"lint":
   {
     "executor": "nx:run-commands",
     "options":
       {
-        "commands": [{ "command": "helm lint", "forwardAllArgs": false }],
+        "commands": [{ "command": "lint", "forwardAllArgs": false }],
```
Comment (nandorszentpeteri) [inline docs/nx/next/init-deployment.md:106]: It's still running the helm-lint command, but please double check

**Review by nandorszentpeteri — CHANGES_REQUESTED** [review]

File: docs/nx/next/init-deployment.md [hunk 2]
```diff
@@ -87,38 +90,40 @@ As a rule of thumb, each task here references a target execution via Nx defined
...
         "cwd": "apps/myapp/build/helm",
```
Comment (nandorszentpeteri) [inline docs/nx/next/init-deployment.md:107]: I missed this one initially, but the cwd has to point to the helm chart, in the code currently `${helmProjectPath}/build/helm`

File: docs/nx/next/init-deployment.md (hunk 1 above)
Comment (MetecanAydin) [reply docs/nx/next/init-deployment.md:106]: command is `helm lint` yes, under target `lint` - updated

File: docs/nx/next/init-deployment.md (hunk 2 above)
Comment (MetecanAydin) [reply docs/nx/next/init-deployment.md:107]: true, updated

**Review by nandorszentpeteri — APPROVED** [review]

File: docs/nx/next/init-deployment.md (hunk 1 above)
Comment (Danphillipz) [inline docs/nx/next/init-deployment.md]: Helm is still in this command
```suggestion
        "commands": [{ "command": "helm lint", "forwardAllArgs": false }],
```

File: docs/nx/next/init-deployment.md (hunk 2 above)
Comment (Danphillipz) [reply docs/nx/next/init-deployment.md]: @MetecanAydin - Have you pushed this update?

Comment (MetecanAydin) [reply docs/nx/next/init-deployment.md]: @Danphillipz yes, should be there

File: docs/nx/next/init-deployment.md (hunk 1 above)
Comment (MetecanAydin) [reply docs/nx/next/init-deployment.md]: yeah this section is outdated. In the actual changes diff, I can see it is correct 😂 
image

**Pull request merged**